<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTicket</class>
 <widget class="QDialog" name="AddTicket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-size: 10pt;
}

QDialog {
	
	background-color: rgb(0, 0, 0);
}

QLabel {
	color: rgb(255, 246, 0);
}

QLineEdit, QDateEdit {
	color: rgb(255, 246, 0);
	
	background-color: rgb(0, 0, 0);
	border: 1px solid #fff600;
}

QPushButton {
	background-color: rgb(0, 0, 0);
	color: rgb(255, 246, 0);
	border: 3px solid #fff600;
}

QPushButton:hover {
	background-color: #fff600;
	color: black;
}

#frame_2 {
	
	background-color: rgb(255, 255, 255, 80);
	border-radius: 4px;
}
</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>381</width>
     <height>296</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblDate">
        <property name="text">
         <string>Date:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dtDate"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="lblNo">
        <property name="text">
         <string>No:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtNo"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lblSource">
        <property name="text">
         <string>Source:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtSource"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lblDest">
        <property name="text">
         <string>Destination:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtDest"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="lblPassName">
        <property name="text">
         <string>Passenger First Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtFName"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblPassFamily">
        <property name="text">
         <string>Passenger Last Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtLName"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="lblPassFatherName">
        <property name="text">
         <string>Passenger Father Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPassFatherName"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="lblPassBirthDate">
        <property name="text">
         <string>Passenger Birth Date:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dtBirthDate"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="lblPassNationalCode">
        <property name="text">
         <string>Passenger National Code:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtNationalCode"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="btnSubmit">
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
